docs/fedora-releases-guide/index.md

```diff
@@ -253,3 +253,28 @@ you can override this behaviour by specifying
 `allowed_builders` in the [job configuration](/docs/configuration/downstream/bodhi_update).
 
 For a configuration example and retriggering, see [dist-git onboarding](./dist-git-onboarding.md#bodhi-update-job).
+
+
+# Permission Handling
+
+The downstream automation is a chain of jobs: propose downstream/pull from upstream -> koji build -> bodhi update.
+
+This chain can break if Packit does not find the right permissions.
+
+### Propose downstream/Pull from upstream -> Koji build
+
+A Koji build can be triggered by a merged pull request, a direct push or a comment.
+
+* If you only merge Packit created pull requests, you don't need to specify other settings.
+* If you create your own pull request and you want Packit to automatically trigger a Koji build when you merge it you need to be in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list, even when you are reusing Packit commits.
+On the other side, if you want to break the chain because, as an example, you want to build the package by yourself in side-tags then you can create a new pull request, re-using the Packit commits, and you have to be sure you are not in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list.
```

Comment:
am I mistaken or this is the use-case that actually doesn't work now 100% because of packit/packit-service#2359 ?

Reply:
Yes, at the moment we could have problems in both cases (when they want to trigger a build and when they don't) if they create a PR reusing the Packit commits. And unfortunately, the music user yesterday showed us that we also have a problem with direct pushes if the commits are taken from a Packit branch with a related PR.

Reply:
thanks for confirming, then I am thinking what would be the best, whether waiting for merging this PR after the issue is resolved, or merging  now and mentioning the issue until it is fixed

Reply:
I would probably mention the issue, as it can take some time before the change makes it to the production src.fp.o instance.